lib/client-options.js

```diff
@@ -20,7 +20,7 @@ function defaultOptions () {
       addressResolution: new addressResolution.AddressTranslator()
     },
     queryOptions: {
-      consistency: types.consistencies.one,
+      consistency: types.consistencies.localQuorum,
```

Comment:
:+1: on making explicit in release notes.  This change will be incorporated into the first RC or GA release of driver 3.0 so it will be associated with a major version bump.